src/components/ThoughtAnnotation.tsx

```diff
@@ -253,20 +257,28 @@ const ThoughtAnnotation = React.memo(
             // Match .editable-annotation-text padding-left.
             // Add 0.5em to account for the superscript.
             // TODO: Add space for dynamic superscript. This is currently only correct for single digit superscript.
-            marginRight: showSuperscript ? '-0.833em' : '-0.333em',
-            paddingRight: showSuperscript ? '0.833em' : '0.333em',
+            marginRight: showSuperscript ? '-0.833em' : 0,
+            paddingRight: showSuperscript ? '0.833em' : 0,
           }}
         >
-          <span className='editable-annotation-text' style={style} dangerouslySetInnerHTML={{ __html: textMarkup }} />
-          {
-            // do not render url icon on root thoughts in publish mode
-            url && !(publishMode() && simplePath.length === 1) && <UrlIconLink url={url} />
-          }
-          {email && <EmailIconLink email={email} />}
-          {
-            // with real time context update we increase context length by 1 // with the default minContexts of 2, do not count the whole thought
-            showSuperscript ? <StaticSuperscript n={numContexts} style={style} /> : null
-          }
+          <span
+            className={classNames({ 'editable-annotation-text': true, url: url, active: isEditing })}
```

Comment:
The `.child` ancestor already has `.editing` applied to it if editing. Let's use that instead of introducing the `active` class.

Reply:
Ah, that's what I was looking for. Thanks!